spec/helpers/physical_storage_helper/textual_summary_spec.rb

```diff
@@ -58,9 +58,9 @@
       expect(subject.title).to eq('Properties')
     end
 
-    it 'shows 8 fields' do
+    it 'shows 9 fields' do
       expect(subject.items).to be_kind_of(Array)
-      expect(subject.items.size).to eq(8)
+      expect(subject.items.size).to eq(9)
```

Comment:
Not a reflection on your changes at all, but this seems like a pointless test.  Better to check the contents than a whole test checking the number of fields.